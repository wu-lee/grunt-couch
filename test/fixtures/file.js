module.exports = {
    foo: {
        bah: {
            title: 'Bah',
            list: ['Uno', 'Duo']
        },
    },
};

