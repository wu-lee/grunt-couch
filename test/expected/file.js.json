{
  "docs": [
    {
      "foo": {
        "bah": {
          "title": "Bah",
          "list": [
            "Uno",
            "Duo"
          ]
        }
      }
    }
  ]
}
